### PR TITLE
Add deploy clone script

### PR DESCRIPTION
**Pull request by iHiD**
This should push to the `website-deployer` clone when in the v3-website repo, and deploy when in the `website-deployer` repo

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -211,3 +214,22 @@ jobs:
...
+    if: github.repository == "exercism/v3-website"
+
+    name: Replicate to exercism/website-deployer
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+        env:
+          WEBSITE_DEPLOYER_CLONE_KEY: ${{ secrets.WEBSITE_DEPLOYER_CLONE_KEY }}
```
Comment (ErikSchierboom) [inline .github/workflows/deploy.yml]: Shouldn't this be part of the step below?

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -24,48 +27,48 @@ jobs:
...
         # https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#overriding-automatic-language-detection
 
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
-      with:
-        # We must fetch at least the immediate parents so that if this is
-        # a pull request then we can checkout the head.
-        fetch-depth: 2
+      - name: Checkout repository
```
Comment (ErikSchierboom) [inline .github/workflows/codeql-analysis.yml:35]: The changes in these file were done by my linter

File: .github/workflows/replicate.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    # TODO: Change this to website when merging
+    if: github.repository == 'exercism/v3-website'
+
+    name: Replicate to exercism/website-deployer
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```
Comment (ErikSchierboom) [inline .github/workflows/replicate.yml:21]: This is required to checkout the entire history of the branch and not just the latest commit

**Comment by iHiD** [comment]
Nice work. Thanks! I'll get the custom runner set back up in the next day or so.

**Pull request merged**